controllers/common/common.go

```diff
@@ -206,16 +209,70 @@ func HasValidPlacementRef(pb *policiesv1.PlacementBinding) bool {
 	}
 }
 
-// GetDecisions returns the placement decisions from the Placement or PlacementRule referred to by
-// the PlacementBinding
-func GetDecisions(
+// The placementDecisionGetter is an implementation of the cluster PlacementDecisionGetter
+type placementDecisionGetter struct {
+	c client.Client
+}
+
+// List the PlacementDecisions using the method signature from the cluster PlacementDecisionGetter
+// interface. (The signature was originally written to retrieve from a cache, so we need to convert
+// the returned values to an array of pointers.)
+func (pd placementDecisionGetter) List(
+	selector labels.Selector, namespace string,
+) ([]*clusterv1beta1.PlacementDecision, error) {
+	pdList := &clusterv1beta1.PlacementDecisionList{}
+	pdPtrList := []*clusterv1beta1.PlacementDecision{}
+	lopts := &client.ListOptions{
+		LabelSelector: selector,
+		Namespace:     namespace,
+	}
+
+	err := pd.c.List(context.TODO(), pdList, lopts)
```

Comment:
Why this need context.TODO()? why not pass context?

Reply:
It's a good point. This `List()` function is buried inside of the `placementDecisionLister` interface, which should be implemented using a `cache.Indexer` instead of this `List()`, so I need to look into making that update..